src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const avarageTime = 3;
+  const longTime = 7;
+  const costOfDay = 40;
+  const avarageTimeDiscount = 20;
+  const longTimeDiscount = 50;
+  const summ = days * costOfDay;
```

Comment:
The variable name 'summ' is not descriptive enough. A better name could be 'totalCost'.